recipe/session/session_test.go

```diff
@@ -657,7 +658,8 @@ func TestWhenAntiCsrfIsDisabledFromSTcoreNotHavingThatInInputToVerifySessionIsFi
 		},
 		RecipeList: []supertokens.Recipe{
 			Init(&sessmodels.TypeInput{
-				AntiCsrf: &customAntiCsrfVal,
+				AntiCsrf:     &customAntiCsrfVal,
+				CookieSecure: &True,
```

Comment:
why is this modified?

Reply:
Test was failing.

Reply:
why?

Reply:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/650429/174253591-8fdb15a7-7f60-4a23-a4a2-ce732c9dce8f.png">

This combination is not allowed. Do you prefer this over secure cookie?

<img width="343" alt="image" src="https://user-images.githubusercontent.com/650429/174254473-41d46550-2683-43d7-a958-a104ae1f88ef.png">